docs/eks-v3-migration.md

```diff
@@ -0,0 +1,152 @@
+  cluster: cluster,
+  operatingSystem: eks.OperatingSystem.AL2,
+  instanceProfile: instanceProfile,
+});
+const export oldAsgName = oldNG.autoScalingGroup.name;
+
+const newNG = new eks.NodeGroupV2("new", {
+  cluster: cluster,
+  operatingSystem: eks.OperatingSystem.AL2023,
+  instanceProfile: instanceProfile,
+});
+
+// Allow all traffic between the old & new NG
+const oldToNew = new aws.vpc.SecurityGroupIngressRule("oldToNew", {
+    securityGroupId: oldNG.nodeSecurityGroup.id,
+    referencedSecurityGroupId: newNG.nodeSecurityGroup.id,
+    fromPort: 0,
+    ipProtocol: "-1",
+    toPort: 0,
+});
+const newToOld = new aws.vpc.SecurityGroupIngressRule("newToOld", {
+    securityGroupId: newNG.nodeSecurityGroup.id,
+    referencedSecurityGroupId: oldNG.nodeSecurityGroup.id,
+    fromPort: 0,
+    ipProtocol: "-1",
+    toPort: 0,
+});
+```
+
+2. Find the nodes of the old node group. First take a note of the name of the auto scaling group associated with that node group and then run the following AWS CLI command, replacing `$ASG_GROUP_NAME` with the actual name of the auto scaling group:
+
+```bash
+aws ec2 describe-instances --filter "Name=tag:aws:autoscaling:groupName,Values=$ASG_GROUP_NAME" \
+    | jq -r '.Reservations[].Instances[].PrivateDnsName'
+```
+
+3. Drain each of the nodes of the old node group one by one. This will mark the nodes as unschedulable and gracefully move pods to other nodes. For more information have a look at this article in the [kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/).
+
+```bash
+kubectl drain $NODE_NAME --ignore-daemonsets --delete-emptydir-data
+```
+
+4. The old nodes are now empty and can be safely shut down. Remove them from your pulumi program and run `pulumi up`
+
+## aws-auth ConfigMap Deprecation
+
+AWS introduced a new method for granting IAM principals access to Kubernetes resources called [Access Entries](https://docs.aws.amazon.com/eks/latest/userguide/grant-k8s-access.html#authentication-modes). In contrast to the existing approach using the aws-auth ConfigMap, this solely relies on AWS resources for managing Kubernetes auth.  
+Recently the aws-auth ConfigMap has been deprecated in favor of Access Entries.  
+You can start using Access Entries with your cluster by changing the `authenticationMode` to `API`.  
+A step-by-step migration guide can be found [here](https://github.com/pulumi/pulumi-eks/blob/master/docs/authentication-mode-migration.md).
+
+At this time we recommend users to create all new clusters with the authentication mode `API.`  
+More details about this can be found [here](https://docs.aws.amazon.com/eks/latest/userguide/grant-k8s-access.html).
+
+## Nodejs SDK breaking changes
+
+Up until this point the provider exposed a hand-written nodejs SDK instead of using the multi-language-component approach that was used for the other SDKs like Python. This caused inaccurate documentation as well as an increased maintenance effort.
```

Comment:
Drop the maintenance effort, not relevant to users. Perhaps something along the lines of:

> Node SDK is updated to use the state of the Pulumi tooling, improving stability, documentation and security.  The update requires the following changes in programs: